### PR TITLE
DEVDOCS-2554: Updated links, added transition sentence

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2554](https://jira.bigcommerce.com/browse/DEVDOCS-2554)

This PR replaces [PR 980](https://github.com/bigcommerce/dev-docs/pull/980).

## What changed?
* Updated On This Page
* Added transition sentence
* Updated Resources links

### CODE REVIEW

File: docs/api-docs/customers/customers-subscribers-overview.md [hunk 1]
```diff
@@ -213,28 +222,27 @@ The Subscribers API allows you to manage subscribers who have signed up for the
...
+- [Customer Groups](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customer-groups/getallcustomergroups) (Customers V2 API)
+- [Password Validation](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customer-passwords/validatecustomerpassword) (Customers V2 API)
+- [Password Confirmation](https://developer.bigcommerce.com/api-reference/store-management/customers-v2/customers/createanewcustomer) (Customers V2 API)
+- [Storefront Subscriptions](https://developer.bigcommerce.com/api-reference/storefront/storefront-subscriptions)
+- [Subscribers API](https://developer.bigcommerce.com/api-reference/store-management/subscribers)
 
 ### Webhooks
-- [Customers](https://developer.bigcommerce.com/api-docs/getting-started/webhooks/webhook-events#webhook-events_customer)
-
+- [Customers](https://developer.bigcommerce.com/api-docs/store-management/webhooks/events#customer)
```
Comment (traci707) [inline docs/api-docs/customers/customers-subscribers-overview.md:248]: The link takes you to a sub-section called "Customer"
Maybe you should change it to 
- [Customer](https://developer.bigcommerce.com/api-docs/store-management/webhooks/events#customer)

File: docs/api-docs/customers/customers-subscribers-overview.md [hunk 2]
```diff
@@ -213,28 +222,27 @@ The Subscribers API allows you to manage subscribers who have signed up for the
 
 **Which API should I use?**
 
-Use the V3 Customers API when possible as we will eventually begin the process of deprecating the existing V2 Customers API. 
+When possible, use the V3 Customers API as BigCommerce will eventually deprecate the existing V2 Customers API. 
 
 **How can I validate customer passwords?**
 
-Password validation is only available on V2 Customers API. Validation will return a boolean. The V3 Customers API can reset a customer's password or input a new password. 
+Password validation is only available on the V2 Customers API. Validation will return a boolean. The V3 Customers API can reset a customer's password or input a new password.
```
Comment (traci707) [inline docs/api-docs/customers/customers-subscribers-overview.md:229]: The sentence "Validation will return a boolean." doesn't seem to belong here.

**Comment by ektravel** [comment]
@uwikaiddi PR 986 is ready for review.

File: docs/api-docs/customers/customers-subscribers-overview.md [hunk 3]
```diff
@@ -98,41 +99,48 @@ A customer makes a purchase on a store and creates an account. The customers obj
...
 
-[Customer form field values](https://developer.bigcommerce.com/api-reference/store-management/customers-v3/customer-form-field-values/) are fields on either the customer address or customer signup that accept any string data. An example of a signup field can have a customer input a wholesaler ID or, for an address field, have them input any special delivery instructions. For more information about creating form fields, see [Account Signup Fields](https://support.bigcommerce.com/s/article/Editing-Form-Fields#account-fields). 
+[Customer form field values](https://developer.bigcommerce.com/api-reference/store-management/customers-v3/customer-form-field-values/) are fields on either the customer address or customer signup that accept any string data. For example, a signup field can have a customer input a wholesaler ID or, for an address field, have them input any special delivery instructions. For more information about creating form fields, see [Account Signup Fields](https://support.bigcommerce.com/s/article/Editing-Form-Fields#account-fields).
```
Comment (uwikaiddi) [inline docs/api-docs/customers/customers-subscribers-overview.md:114]:
```suggestion
[Customer form field values](https://developer.bigcommerce.com/api-reference/store-management/customers-v3/customer-form-field-values/) are fields on either the customer address or customer sign-up that accept any string data. For example, a sign-up field can have a customer input a wholesaler ID or, for an address field, have them input any special delivery instructions. For more information about creating form fields, see [Account Signup Fields](https://support.bigcommerce.com/s/article/Editing-Form-Fields#account-fields). 

```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**